tests/README.md

```diff
@@ -0,0 +1,138 @@
+# Substrait Test Format
+
+This document describes the format for Substrait scalar test files.
+A test file consists of the following elements:
+
+1. Version declaration
+2. Optional include statements
+3. One or more test groups, each containing one or more test cases
+
+## Syntax
+
+### Version Declaration
+The version declaration must be the first line of the file. It specifies the version of the test file format. The version declaration must be in the following format:
+```
+### SUBSTRAIT_SCALAR_TEST: V1
+```
+
+### Include Statements
+Include statements should have at least one include statement. The include statement specifies the path to substrait extension functions. The include statement must be in the following format:
+```
+### SUBSTRAIT_INCLUDE: /extensions/functions_aggregate_approx.yaml
```

Comment:
1. Functions in the same extension yaml file will differ in the signature (function name or argument types)
2. In case of a conflict with function from two different extensions is to avoid the conflict by having test cases for such functions in separate test files.